style.css

```diff
@@ -902,3 +928,177 @@ body {
     top: 65%;
   }
 }
+
+@media only screen and (min-width: 768px) {
+  #nav {
+    display: none;
+  }
+
+  #item_nav {
+    right: 150px;
+  }
+
+  #recentwork_text_container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #featured_img_container {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+  }
+
+  #img_placeholder {
+    height: 450px;
+  }
+
+  #frame44 {
+    width: calc(100% + 120px);
+    height: 1px;
+    position: relative;
+    left: 60px;
+  }
+
+  #frame43 {
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px;
+    height: 0;
+    position: relative;
+  }
+
+  #recentwork {
+    padding: 120px 120px 120px 0;
+  }
+
+  #recentwork_divider {
+    width: 500px;
+  }
+
+  #action_btn {
+    display: none;
+  }
+
+  #card1 {
+    display: none;
+  }
+
+  #card1_data {
+    display: none;
+  }
+
+  #action_btn1 {
+    top: 1px;
+    width: 100%;
+    z-index: 1;
+  }
+
+  #card_data {
+    margin: 120px 0 0 0;
+  }
+
+  #card_title {
+    display: none;
+  }
+
+  #desktop_card_img {
+    display: block;
+  }
+
+  #desktop_card_title {
+    display: block;
+    width: 295px;
+    height: 81px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  #card_works {
+    width: 369px;
+    height: 435px;
+  }
+
+  .desktop_delete_first {
+    display: none;
+  }
+
+  #about_me {
+    top: 1850px;
+    width: 100%;
+    left: 0;
+    align-items: flex-start;
+    padding: 0 0 0 24px;
+  }
+
+  #about_me::after {
+    content: "";
+    border-style: none none solid none;
+    border-radius: 1px;
+    border-color: #fbf8f7;
+    position: absolute;
+    bottom: 0;
+    left: 10%;
+    height: 1px;
+    width: 80%;
+    margin: 0 0 0 -40px;
+  }
+
+  #me_art {
+    background-image: url("./assets/illustration_last_desktop.svg");
+    width: 586px;
+    height: 354px;
+    top: 70px;
+  }
+
+  #desktop_about_me_text {
+    margin: 0 0 0 40px;
+  }
+
+  #me_rectangle {
+    left: 0;
+    height: 1200px;
+  }
+
+  #me_vector {
+    display: none;
+  }
+
+  #skill {
+    top: 2400px;
+  }
+
+  #skill_parent {
+    display: flex;
+    flex-direction: row;
+    gap: 25%;
+    justify-content: center;
+  }
+
+  #form {
+    top: 3850px;
+    flex-direction: row;
+    width: 100%;
+    align-items: baseline;
+  }
```

Comment:
- [ ] Please you may need to increase the gap between the two children elements according to the template. More so, there should be margin before the `get in touch` button. Please kindly effect this. 👍 

## Your Design
![Screenshot from 2022-11-25 09-04-14 (1)](https://user-images.githubusercontent.com/103386183/206680414-92e4accd-28da-470c-87a0-91e5f78b03c3.png)